Dockerfile

```diff
@@ -1,17 +1,25 @@
 ARG BUILDX_VERSION=0.6.0
+ARG DOCKER_CONFIG=/env_configs/.docker
+ARG BUILDX_BIN=$DOCKER_CONFIG/cli-plugins/docker-buildx
 
 FROM docker/buildx-bin:$BUILDX_VERSION as buildx_bin
 
-FROM docker:latest as buildx_image
+FROM alpine:3.14.0 as buildx_strip
 
-ARG DOCKER_CONFIG=/env_configs/.docker
+ARG BUILDX_BIN
 
-ENV DOCKER_CONFIG=$DOCKER_CONFIG \
-    DOCKER_CLI_EXPERIMENTAL="enabled"
+COPY --from=buildx_bin /buildx $BUILDX_BIN
+RUN apk add -U binutils && strip $BUILDX_BIN
 
-WORKDIR $DOCKER_CONFIG/cli-plugins
+FROM docker:20.10.7 as buildx_image
```

Comment:
Can we use args for the docker/alpine versions? (just in case we _do_ want to update them in the future easily)

Reply:
Oh, yes! I thought you'd never ask! ;-)

Update coming up.